docs/en/compute-edition/32/rn/release-information/known-issues-32.adoc

```diff
@@ -0,0 +1,22 @@
+== 31.xx Known Issues
```

Comment:
@rcaballeromx did you want to update the version number to O'Neal here as well!

Reply:
Thank you Divya but I created PR #268 to address the release note updates.